gilded_rose.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+class GildedRose(object):
+
+    def __init__(self, items):
+        self.items = items
+
+    def update_quality(self):
```

Comment:
Тип возращаемого значения отсутствует
```suggestion
    def update_quality(self) -> None:
```